pkg/api/operators/apis/experiment/v1alpha2/experiment_defaults.go

```diff
@@ -0,0 +1,27 @@
+// +build !ignore_autogenerated
+
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+// Code generated by main. DO NOT EDIT.
+
+package v1alpha2
+
+func (e *Experiment) Default() {
```

Comment:
Many be SetDefaults?

Reply:
I can update it (in fact, I see many kube project use Default(), too).